clar2wasm/tests/standard/unit_tests.rs

```diff
@@ -4201,3 +4201,75 @@ fn utf8_to_string_utf8_invalid() {
         check_invalid_conversion(s.as_bytes(), s.chars().count() as i32 - 1);
     }
 }
+
```

Comment:
Would be nice to also have a unit test for `compare_clarity_names`.

Reply:
That's an internal function for `bsearch-clarity-name`, hence not exported. It's tested "transitivityly" with the tests for this function.

We can always add the export inside the test manually after a call to `load_stdlib` but I'm not fond of the idea: this is testing an implementation detail that could change with a refactor.